packages/syntax-highlighter/canonical.js

```diff
@@ -0,0 +1,60 @@
+const modeAliases = {
```

Comment:
It might be kind of tedious to maintain this list along with `uppercase.js` instead of just a single `supportedLanguages` file, but also I think the object management, and cycles, that'll need to be done to make a single datastore work for both of these usecases (canonical and uppercase) outweighs just loading another 60 line JS file.

Reply:
Yeah the manual maintenance is a bit of a pain, but I agree: worth it compared to the alternative!